docs/source/python/api/compute.rst

```diff
@@ -479,6 +494,7 @@ Structural Transforms
 .. autosummary::
    :toctree: ../generated/
 
+   cumulative_sum
```

Comment:
Why is this added here?

Reply:
This was before the addition of the "Cumulative Functions" section. It can be removed from here now.